.github/workflows/ci.yml

```diff
@@ -152,9 +153,18 @@ jobs:
           </configurable_settings>
           EOF
 
+          # Create the symbolic link to pass `check_concinnity` tests.
+          ln -s $GITHUB_WORKSPACE /opt/lmi/src/lmi
```

Comment:
Do you know what would be the minimal change to allow us to get rid of this one? If it's something simple, perhaps we could ask Greg to do it. Otherwise I'll just merge this as is. TIA!

Reply:
The problem occured when `*.sh` scripts called with absolute paths:
```bash
# lmi_setup_43.sh(100)
. /opt/lmi/src/lmi/set_toolchain.sh 

# lmi_setup_44.sh(34)
/opt/lmi/src/lmi/nychthemeral_test.sh || true

# nychthemeral_test.sh(226)
/opt/lmi/src/lmi/test_schemata.sh 2>&1 \

# nychthemeral_test.sh(255)
srcdir=. datadir=. /opt/lmi/src/lmi/mst_to_xst.sh
```

Reply:
OK, this probably won't be changed in the near future, so I'll merge this PR in its current state, thanks.